docs/reference/eql/eql-search-api.asciidoc

```diff
@@ -534,7 +534,12 @@ in ascending order.
         },
         "sort": [
           1607252647000
-        ]
+        ],
+        "fields": {
```

Comment:
@jrodewig is there a way to ignore the "fields" field? This data is used internally and it might be that in the future, we'll remove it from the final response (so only `_source` is included).

Reply:
You can use the `filter_path` query param to exclude `fields` from the output. 
You can add the query param using a `//TEST` comment so it's not visible to users.

This should fail gracefully. The request won't return an error if `fields` is later removed from the response.

Here's an example with a basic query:
```
[source,console]
----
GET /sec_logs/_eql/search
{
  "query": """
    process where process.name == "cmd.exe"
  """
}
----
// TEST[s/search/search\?filter_path\=\-\*\.events\.\*fields/]
```

A sequence query will need to use a different filter because the response format is different.

```
[source,console]
----
GET /my_index/_eql/search
{
  "query": """
    sequence by agent.id
      [ file where file.name == "cmd.exe" and agent.id != "my_user" ]
      [ process where stringContains(process.path, "regsvr32") ]
  """
}
----
// TEST[s/search/search\?filter_path\=\-\*\.sequences\.events\.\*fields/]
```

Reply:
For some reason this didn't seem to work consistently so I left things in a bit of a mixed state, namely the `fields` tag is still in there but I also added the `TESTRESPONSE`; the idea being that the user should not depend on whether `fields` (or even `sort`) are in the response.
Can you please take a look at it after this PR gets merged?

Reply:
Sure thing. Thanks!